modules/op-using-pipelines-as-code-with-bitbucket-cloud.adoc

```diff
@@ -120,8 +120,8 @@ spec:
 * The `tkn pac create` and `tkn pac bootstrap` commands are not supported on Bitbucket Cloud.
 
 * Bitbucket Cloud does not support webhook secrets. To secure the payload and prevent hijacking of the CI, {pac} fetches the list of Bitbucket Cloud IP addresses and ensures that the webhook receptions come only from those IP addresses.
-** To disable the default behavior, set the `bitbucket-cloud-check-source-ip key` to `false` in the {pac} config map for the `pipelines-as-code` namespace.
-** To allow additional safe IP addresses or networks, add them as comma separated values to the `bitbucket-cloud-additional-source-ip` key in the {pac} config map for the `pipelines-as-code` namespace.
+** To disable the default behavior, set the `bitbucket-cloud-check-source-ip key` to `false` in the `TektonConfig` custom resource, under `pipelinesAsCode` and `settings`.
```

Comment:
same comment as earlier, recommend instead saying "in the `pipelinesAsCode.settings` spec"